src/impl/sbcl-vops.lisp

```diff
@@ -0,0 +1,292 @@
+;;;; sbcl-vops.lisp
+;;;;
+;;;; Author: Cole Scott
+;;;;         
+;;;; Collaborators: Jussi Kukkonen
+;;;;                Robert Smith
+
+(defpackage #:qvm-intrinsics
+  (:use #:cl #:sb-ext #:sb-c))
+
+(in-package #:qvm-intrinsics)
+
+#+avx2
```

Comment:
see note about splitting above. let's try to keep reader conditionals out of these files